pegasus-common/src/main/java/com/linkedin/internal/common/SchemaDirLocation.java

```diff
@@ -0,0 +1,36 @@
+/*
+   Copyright (c) 2020 LinkedIn Corp.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package com.linkedin.internal.common;
+
+/**
+ * Directories within resource jar file that holds different types of pegasus schemas. Ex. Data or Extensions
+ *
+ * @author Aman Gupta
+ */
+public enum SchemaDirLocation
```

Comment:
"Schema dir location" seems a little confusing and kinda redundant. It seems to me that it's more of a schema dir _"type"_ or schema dir _"name"_. Also, I would prefer that the class name contains the word "Directory" rather than the abbreviated "Dir"

So perhaps something like:
- `SchemaDirectory`
- `SchemaDirectoryType`
- `SchemaDirectoryName`